lib/public_key/test/pkits_SUITE.erl

```diff
@@ -198,18 +282,31 @@
 -define(NIST5, "2.16.840.1.101.3.2.1.48.5").
 -define(NIST6, "2.16.840.1.101.3.2.1.48.6").
 
+-define(NIST1_OID, {2,16,840,1,101,3,2,1,48,1}).
```

Comment:
This is because the standard test suite  defines it like that,  so I have accepted the string format as input. But the tuple format is what the ASN.1 will generate.